### ISSUE TITLE
Extremely fat repo - takes several minutes to clone

### ISSUE THREAD

**Issue by oesteban** [poldracklab/nondefaced-detector#14]
I believe we have too much binary data in the repo. We should use datalad for those and store the binary data on OSF or similar.

**Comment by shashankbansal6** [comment]
Fixed in https://github.com/poldracklab/nondefaced-detector/commit/9214d1c6cd726f6b4edf33df9ccc5ca46da88232

**Comment by shashankbansal6** [comment]
Okay so it looks like this will be much more involved than I initially thought. My first guess was that the bulk of the fat was because of the pretrained_weights we initially added with the repo but I removed them completely from the entire history and that still didn't seem to work.
```
git filter-branch --force --index-filter \
'git rm -rf --ignore-unmatch nondefaced_detector/models/pretrained_weights' \
--prune-empty --tag-name-filter cat -- --all
```
This leads me to believe that much of the glut is coming from our initial commits during the time Avinash was working on this before I rebuilt this repo as nondefaced-detector. I am going to 
1. either rebase all of those commits
2. create another repo and cherry pick commits to copy over
